medusa/tv/series.py

```diff
@@ -1223,7 +1223,7 @@ def load_from_indexer(self, tvapi=None):
         self.classification = getattr(indexed_show, 'classification', 'Scripted')
         self.genre = getattr(indexed_show, 'genre', '')
         self.network = getattr(indexed_show, 'network', '')
-        self.runtime = int(getattr(indexed_show, 'runtime', 0))
+        self.runtime = int(getattr(indexed_show, 'runtime', 0) or 0)
```

Comment:
Because you don't know the actual time.  For logic like when to search you can easily do `self.runtime or 60` when the runtime is falsey (i.e. `None` or `0`).  Conversely there's no easy way to do the reverse.  If you assign a runtime of `60` the only way to know its a fake time (or default time, if you prefer) is to check the indexer again.